docs/data-sources/scalr_provider_configurations.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: "scalr"
+page_title: "Scalr: scalr_provider_configurations"
```

Comment:
@DayS1eeper @emocharnik Please tell me more about the use case for this data source

Reply:
The main idea behind having listing dat-source is to fetch all required at once (e.g for BU-1) and link it to respective environment

Reply:
Now to achieve that users do multiple requests to the api server

Reply:
@emocharnik ok ty.